webapp/src/ts/services/enketo.service.ts

```diff
@@ -292,12 +292,18 @@ export class EnketoService {
           return Promise.reject(new Error(JSON.stringify(loadErrors)));
         }
         const language = options.language;
+        this.currentForm.langs.$formLanguages.val(language).trigger('change');
+        // re-set the enketo form's language when a DOM node is added/removed to the form
```

Comment:
I think this comment is out of date. If I'm understanding correctly, the `change` event fires when an input in the form is updated by the user, not when a DOM node is added or removed.